README.md

```diff
@@ -131,6 +131,23 @@ To suppress specific warnings from being treated as errors, add a comma-separate
 </Project>
 ```
 
+You can suppress warnings for a specific file by adding `SuppressTSqlWarnings` for the this file:
+
+```xml
+<Project Sdk="MSBuild.Sdk.SqlProj/1.12.0">
+    <PropertyGroup>
+        ...
+    </PropertyGroup>
+
+    <ItemGroup>
+        <Content Include="Procedures\csp_Test.sql">
+            <SuppressTSqlWarnings>71502</SuppressTSqlWarnings>
+        </Content>
+    </ItemGroup>
+</Project>
+```
+
```

Comment:
We should probably explain how this works in combination with the project-level `SuppressTSqlWarnings`. Looking at the code it seems that if a particular warning is suppressed at the project level it will always be suppressed regardless of what is configured at the file level. I think that behaviour is fine, but we should at least explain that in the docs.

Reply:
Sorry but I'm afraid that my English will not give me the opportunity to write a competent explanation of this behavior. this behavior is the same as at standard SSDT project.

Reply:
Perhaps it is sufficient to add the following after the example:

```
> Note: Warnings suppressed at the project level are always applied to every file in the project, regardless of what is configured at the file level.
```